I'm sorry, but an uncaught exception occurred.

Before loading the script.
WindowsError: [Error 267] : u'C:\\Users\\USER\\Documents\\\uce74\uce74\uc624\ud1a1 \ubc1b\uc740 \ud30c\uc77c\\\uc2e0\uc11d\uae30\ub140\\CaveWoman\\game\\script.rpy\\*.*'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\USER\Downloads\renpy-7.1.3-sdk\renpy\bootstrap.py", line 313, in bootstrap
    renpy.main.main()
  File "C:\Users\USER\Downloads\renpy-7.1.3-sdk\renpy\main.py", line 276, in main
    for fn in os.listdir(dir):
WindowsError: [Error 267] : u'C:\\Users\\USER\\Documents\\\uce74\uce74\uc624\ud1a1 \ubc1b\uc740 \ud30c\uc77c\\\uc2e0\uc11d\uae30\ub140\\CaveWoman\\game\\script.rpy\\*.*'

Windows-7-6.1.7601-SP1
Ren'Py 7.1.3.1092
 
Fri Jan 25 20:20:47 2019
